sql/src/main/java/io/crate/metadata/sys/ClassifiedHistograms.java

```diff
@@ -54,6 +57,33 @@ public Histogram histogram() {
         public Classification classification() {
             return classification;
         }
+
+        long sumOfDurations() {
+            return sumOfDurations;
+        }
+    }
+
+    private static class Metrics {
+
+        private final ConcurrentHistogram histogram;
+        private final AtomicLong sumOfDurations = new AtomicLong();
```

Comment:
Would it make sense to use a `LongAdder` here instead?

Reply:
ah yes, I keep forgetting about this ! great suggestion